.travis.yml

```diff
@@ -138,10 +67,14 @@ before_script:
 
 script:
   - export SEQAN3_NO_VERSION_CHECK=1
+  - export CCACHE_COMPRESS=true
+  - export CCACHE_COMPRESSLEVEL=6
+  - export CCACHE_MAXSIZE=5G
   - make -k -j2
   - |
-    if test coverage != "${BUILD}"; then
-      if [[ "${BUILD}" =~ ^(snippet)$ ]]; then
+    if [[ "${{ matrix.build }}" =~ ^(coverage)$ ]]; then
+      : ; else
```

Comment:
what happens here?

Reply:
We don't need to run ctest if we are building coverage - running the tests is already part of building the coverage, so I just do nothing (no-op operator `:`).

Reply:
It was `if test coverage != "${BUILD}"; then` before, but I didn't understand what it did until I removed it; I think not mixing different syntaxes probably helps understanding